### DISCUSSION TITLE
Duda de la fecha limite de entrega de la practica 1

### DISCUSSION THREAD

**Discussion by FabiMateosCat** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#246] [Q&A] [answered]
Alguien sabe cual es la fecha Límite de entrega de la practica de abogabot,es que parece que se actualizo?

**Comment by manusanabria** [comment, marked as answer]
Si no mal recuerdo, la fecha límite es el domingo 6 de marzo.

**Reply by FabiMateosCat** [reply]
Gracias!!!

**Reply by ABD2k** [reply]
saben donde la subo?

**Reply by JosePabloGrr** [reply]
En Teams > Semana 1 > Link de Entrega.
![image](https://user-images.githubusercontent.com/16726985/156841516-5253ee46-5d3e-4f55-a9bd-155b512ac9f0.png)

**Comment by villegas97** [comment]
Yo tenía entendido que era hoy, pero vi en Teams que algunos compañeros comentan que se alargo hasta el 6.

**Comment by JuanSalvador73** [comment]
Era el 4, pero se aplazó hasta el 6. Como sea, si la tienes antes mejor, claro, siempre que te sientas a gusto con ello. :D
